core/src/main/java/com/microsoft/applicationinsights/telemetry/MetricTelemetry.java

```diff
@@ -186,17 +195,13 @@ protected MetricData getData() {
     }
 
     private void updateKind() {
```

Comment:
@macrogreg just for my information, does the backend still has the distinction between aggregate and measurement? I thought all of the metrics are now treated as aggregate. Am I not following anything here?